client/src/components/Users/Auth/SignUpCitizen/SignUpCitizen.tsx

```diff
@@ -74,8 +90,10 @@ function SignupCitizen() {
 
   const makeCitySelectOptions = (state: string) => {
     // TODO: get list of cities from state
-    let cities = Array.from(Array(50).keys()).map((num) => <MenuItem value={num}>{num}</MenuItem>);
-    return cities;
+    const citiesArray = ['Seattle', 'Detroit', 'Chicago', 'Nashville'];
+    return citiesArray.map((city) => {
+      return <MenuItem value={city}>{city}</MenuItem>;
+    });
```

Comment:
@rmwerner18 , for now, let's make this field a simple input element similar to zipcode. This will simplify our work.